src/plugin/road_network.cc

```diff
@@ -46,6 +46,9 @@ class RoadNetworkLoader : public maliput::plugin::RoadNetworkLoader {
       const std::map<std::string, std::string>& properties) const override {
     return maliput_osm::builder::RoadNetworkBuilder(properties)();
   }
+  std::map<std::string, std::string> GetDefaultParameters() const override {
+    return maliput_osm::builder::config::GetDefaultParameters();
```

Comment:
Same comment as in its sibling PR. Can we do it here instead of creating a function for it?

Reply:
Done!